clouds/snowflake/libraries/javascript/libs/h3_boundary.js

```diff
@@ -1,6 +1,27 @@
 import { h3IsValid, h3ToGeoBoundary } from '../src/h3/h3_boundary/h3core_custom';
 
+function removeDuplicates (coordinates) {
+    const precision = 0.0000000000001;
+    const uniqueCoordinates = new Set();
+
+    for (const coordinate of coordinates) {
+        let isUnique = true;
+        for (const uniqueCoordinate of uniqueCoordinates) {
```

Comment:
We could compare only with the previous vertex (modulo the number of vertex) and safe a few comparisons, not a big deal for such small polygons, but for 6 coordinates we'd do 12 coordinate comparisons  instead of 30

Reply:
Good point